exchange.lua

```diff
@@ -985,8 +983,8 @@ function ex_methods.sell(self, p_name, ex_name, item_name, amount, rate)
 				return false, log2_err
 			end
 
-			remaining = 0
-			revenue = revenue + remaining * row.Rate
+			break
+			--revenue = revenue + remaining * row.Rate
```

Comment:
I can't see where these are accounted for later. I need remaining to be 0 if it really should be 0, because otherwise the add_order later will fail (in the DB schema you can't add orders for 0 quantity). Also revenue has to be updated so the player will get the correct number of credits at line 997/995